### PR TITLE
feat: optimize chat block peek view style

### PR DESCRIPTION
**Pull request by graydawnc**
[BS-1091](https://linear.app/affine-design/issue/BS-1091/ai-chat-block-打开时追加一个自动滚动行为)

### CODE REVIEW

**Comment by graydawnc** [comment]
* **#8644** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8644?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @donteatfriedrice and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/blocksuite/presets/ai/peek-view/chat-block-peek-view.ts [hunk 1]
```diff
@@ -450,12 +457,18 @@ export class AIChatBlockPeekView extends LitElement {
...
-      if (this._chatMessagesContainer) {
-        this._chatMessagesContainer.scrollTop =
-          this._chatMessagesContainer.scrollHeight;
-      }
-    });
+    requestAnimationFrame(this._scrollMessageContainerToBottom);
+  }
+
+  override updated(changedProperties: PropertyValues) {
+    super.update(changedProperties);
```
[the remark this answers is not in the document]
Comment (forehalo) [reply packages/frontend/core/src/blocksuite/presets/ai/peek-view/chat-block-peek-view.ts]: @donteatfriedrice take a look at this

File: packages/frontend/core/src/blocksuite/presets/ai/peek-view/chat-block-peek-view.ts [hunk 2]
```diff
@@ -450,12 +457,18 @@ export class AIChatBlockPeekView extends LitElement {
...
-          this._chatMessagesContainer.scrollHeight;
-      }
-    });
+    requestAnimationFrame(this._scrollMessageContainerToBottom);
+  }
+
+  override updated(changedProperties: PropertyValues) {
+    super.updated(changedProperties);
+    if (
+      changedProperties.has('chatContext') &&
```
Comment (pengx17) [inline packages/frontend/core/src/blocksuite/presets/ai/peek-view/chat-block-peek-view.ts:466]: I think "scroll to the bottom with incoming message" should only happen if the content is at the bottom of the scroll container. This is the behavior of claude (or perhaps chatgpt web as well?)

**Comment by graydawnc** [comment]
### Merge activity

* **Nov 1, 3:40 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/8644) disabled "merge when ready" on this PR due to: a merge conflict with the target branch; resolve the conflict and try again..

**Pull request closed without merging**